torch/_inductor/config.py

```diff
@@ -1707,6 +1707,21 @@ class triton:
         os.environ.get("TORCHINDUCTOR_ENABLE_TLX_TEMPLATES", "0") == "1"
     )
 
+    proton_profiling: bool = (
+        os.environ.get("TORCHINDUCTOR_TRITON_PROTON_PROFILING", "0") == "1"
+    )
+    proton_output_dir: str = os.environ.get(
+        "TORCHINDUCTOR_TRITON_PROTON_OUTPUT_DIR", ""
+    )
+    # Group CTAs by SM in proton trace files.
+    proton_group_by_sm: bool = (
+        os.environ.get("TORCHINDUCTOR_TRITON_PROTON_GROUP_BY_SM", "1") == "1"
```

Comment:
How many proton configs do you thinks user would want interact with? Do you think it's worth making a more direct path?

Reply:
not sure i get the question? i think there is pretty much a single config anybody would care about at a time. even then these profiles are pretty large and i am wondering what sort of aggregation / summary stats might be more useful than the raw traces. eg, i think not resolving warps might be broadly even more useful, and maybe reordering onto "occupancy slots" rather than distinct traces might be good. and providing avg cta runtime + avg time to launch as overall stats.

tho in principle i could see us also go in the other direction and annotating finer-grained phases of execution. not sure what that would look like in general, though.